core/src/test/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -1576,7 +1586,7 @@ private void removeDisruptionSchemeFromNode(NodeAndClient nodeAndClient) {
         return Collections2.filter(nodes.values(), new DataOrMasterNodePredicate());
     }
 
-    private static final class DataNodePredicate implements Predicate<NodeAndClient> {
+    public static final class DataNodePredicate implements Predicate<NodeAndClient> {
```

Comment:
ok, maybe wait with that until we agree on https://github.com/elastic/elasticsearch/pull/11666/files#r35091594 . I only need the predicates in restartRandomNode anyway.